PyRDF/Spark/Backend.py

```diff
@@ -2,8 +2,8 @@
 
 from PyRDF import DataFrame
 from PyRDF import Node
-from PyRDF.Backends import Base
-from PyRDF.Backends import Utils
+from PyRDF import Base
```

Comment:
I don't think you need to remove `Backends` from `PyRDF`. The internal structure that you have in `PyRDF` does not affect what will appear as user interface in `ROOT.RDF.Distributed`, since there we will do a dynamic injection of what we want where we want. Having `Backends` in `PyRDF` keeps things more tidy.

Reply:
I see. I just thought I would mirror all changes that would go into ROOT. I'll change this quickly

Reply:
My point is, the internal structure of `PyRDF` can stay as we want, whatever makes more sense. What the user will see is not that, but what we want them to see in `ROOT.RDF.Distributed`.